src/docs/architecture.mdx

```diff
@@ -7,8 +7,8 @@ title: Architecture
 ```mermaid
 graph TD
 app[Your Application] --> |SDK| lb{{Load Balancer}}
-lb --> relay
-lb --> sentry_web["Sentry (web)"]
+lb --> |"ingest.sentry.example.com"| relay
```

Comment:
For onpremise and local dev, we usually have the same domain for ingest and the web API. There is an optional setting to add the ingest domain. I'd propose to leave the same domain here. 

Separately, it's Relay that handles the store endpoint, so probably we should add that here instead of web:

```suggestion
lb --> |"sentry.example.com/api/\d+/store/"| relay
```